lib/commands/web.js

```diff
@@ -164,8 +164,8 @@ async function tapWebElementNatively (driver, atomsElement) {
       // use tap because on iOS 11.2 and below `nativeClick` crashes WDA
       const rect = await driver.proxyCommand(`/element/${el.ELEMENT}/rect`, 'GET');
       const coords = {
-        x: rect.x + rect.width / 2,
-        y: rect.y + rect.height / 2,
+        x: parseInt(rect.x + rect.width / 2, 10),
```

Comment:
I'm not quite sure about what we are parsing here

Reply:
Mostly just making calculations from the logs easier. `rect.width / 2` can be a float. Make it into an int. WDA can handle it, it just makes reading what is going on easier.

Reply:
Then Math.floor would be more readable. One usually uses parseInt if the argument is expected to be a string